scripts/newreference.py

```diff
@@ -6,10 +6,12 @@
 
 def new_reference(referencefile, outgenbank, outfasta, gene):
     ref = SeqIO.read(referencefile, "genbank")
+    startofgene = None
+    endofgene = None
```

Comment:
I like this pattern of declaring variables that should be set by subsequent code. We should probably check whether the values of these variables are still `None` after the loop below completes and throw an error in the case that the user has requested a specific gene with `--gene`. I would expect the script to run without error if the user ran it without the `--gene` argument and produce outputs for the full genome.

Reply:
I agree with your assessment:

* If `--gene` argument given, this script should error out. 
* If `--gene` argument missing, then produce outputs for the full genome. 

Fixed in 3400636149912d7ab6c219c3ec2dceca9356bb48 but feel free to suggest better wording in the commit or error message.

Reply:
Thanks for the suggestion here: https://github.com/nextstrain/rsv/commit/3400636149912d7ab6c219c3ec2dceca9356bb48#r139572918

Agree the gene check should be before feature parsing: 61af263cc4e3ba1c8a3c0d3d7e486237514a9805